magit-todos.el

```diff
@@ -84,6 +84,8 @@
 (require 'pcre2el)
 (require 's)
 
+(declare-function 'org-show-entry "org")
```

Comment:
FYI this doesn't seem to be necessary since we `(require 'org)`.  I tested without this and the byte-compiler didn't complain.